src/main/java/uk/gov/ons/ctp/response/casesvc/client/CollectionExerciseSvcClient.java

```diff
@@ -44,6 +45,7 @@
    * @param collectionExerciseId the UUID to search by
    * @return the asscoaited CollectionExercise
    */
+  @Cacheable("collectionexercises")
```

Comment:
Is it worth making `COLLEX_CACHE_NAME` public in `CaseSvcApplication` so that, rather than using a string literal here, we could instead just call `CaseSvcApplication.COLLEX_CACHE_NAME` to avoid errors if we ever change this name?

Reply:
👍

Reply:
@mfcrocker Done